financialaid/templates/review_financial_aid.html

```diff
@@ -0,0 +1,112 @@
+{% extends "base.html" %}
+{% load i18n static %}
+
+{% block title %}{% trans "Review Financial Aid" %}{% endblock %}
+
+{% block content %}
+  {% include "header.html" %}
+  <div class="page-content">
+    <div class="single-column" style="max-width: 1000px;">
+      <p><h4>Financial Aid Submissions</h4></p>
+      <div class="mdl-card">
+        <!-- Sorting buttons -->
+        <div>
+          <strong>Sort by: </strong>
+          <div class="btn-group">
+          {% for field in sort_field_info %}
+            <a href="{% url 'review_financial_aid' program_id=current_program_id status=selected_status %}?sort_by={{ field.sort_field }}"
+               class="btn btn-default btn-sm">
+              {{ field.display }}
+              {% if field.direction_display == "" %}&#9650;{% endif %}
+              {% if field.direction_display == "-" %}&#9660;{% endif %}
+            </a>
+          {% endfor %}
+          </div>
+        </div>
+        <!-- /Sorting buttons -->
+        <!-- Filters -->
+        <div>
+          <select class="form-control" onchange="location = this.options[this.selectedIndex].value;">
+          {% for status, message in financial_aid_statuses %}
+            <option value="{% url 'review_financial_aid' program_id=current_program_id status=status %}"
+                    {% if selected_status == status %}selected{% endif %}>
+              {{ message }}
+            </option>
+          {% endfor %}
+          </select>​
+        </div>
+        <!-- /Filters -->
+        <!-- Pagination buttons -->
+        <div>
+          <span class="step-links">
+          {% if page_obj.has_previous %}
+            <a href="?page={{ page_obj.previous_page_number }}" class="btn btn-default"><</a>
+          {% else %}
+            <button class="btn btn-default disabled"><</button>
+          {% endif %}
+          {% if page_obj.has_next %}
+            <a href="?page={{ page_obj.next_page_number }}" class="btn btn-default">></a>
+          {% else %}
+            <button class="btn btn-default disabled">></button>
+          {% endif %}
+          </span>
+          <br>
+          <span class="current">Page {{ page_obj.number }} of {{ paginator.num_pages }}</span>
+        </div>
+        <!-- /Pagination buttons -->
+        
+        <table class="table">
+          <thead>
+            <tr>
+              <th></th>
+              <th>Name/Location</th>
+              <th>Adjusted Cost</th>
+              <th>Income/Yr.</th>
+              <th>Date Calculated</th>
+              <th></th>
+            </tr>
+          </thead>
+          <tbody>
+          {% for financial_aid in financial_aid_objects %}
+            <tr>
+              <td>
+                  <img src="https://s3.amazonaws.com/odl-micromasters-production/avatar_default.png"
```

Comment:
Please use `{% static 'images/avatar_default.png' %}` instead of linking to AWS directly